scripts/dockerbot/dockerbot.py

```diff
@@ -176,6 +177,17 @@ def cache_files(self):
             if self.requirements_update:
                 self.args['update_pip'] = True
 
+    def get_base_branch(self, pr):
+        token = self.args['gh_token']
+        owner = self.args['gh_owner']
+        repo = self.args['gh_repo']
+        if token:
```

Comment:
So if `token` is not set, this function will return `None`, setting `self.base_branch` to `None` later setting `self.env_details['BASE_BRANCH']` to `None` as well. I'm not saying it's probable that token wouldn't get passed in but this `if token` statement is a bit of a lie :stuck_out_tongue_winking_eye: shielding us from an exception but not from problems further down the road. I'm thinking - can we keep this and change `self.base_branch = self.get_base_branch(self.args['pr'])` into `self.base_branch = self.get_base_branch(self.args['pr']) or self.base_branch` so that it will be always set to something?